modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -556,6 +556,25 @@ Replicas:          1 desired | 1 updated | 1 total | 0 available | 1 unavailable
   hello-world-77f5897949-z54gb   Pending
 ----
 
+== Auditing rtfctl Operations
+
+The `rtfctl` command line tool (from vX.X.X and above) audits the `rtfctl` operations on the cluster. Helping cluster administrators and you to track operations that impact Runtime Fabric core software.
```

Comment:
It seems that a connector is missing.
The sentence may read:
"The `rtfctl` command line tool (from vX.X.X and above) audits the `rtfctl` operations on the cluster and it helps cluster administrators and you to track operations that impact Runtime Fabric core software."